packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.js

```diff
@@ -19,6 +19,13 @@ import expandFacet from '@salesforce/label/c.quantic_ExpandFacet';
 /** @typedef {import("coveo").DateFacetValue} DateFacetValue */
 /** @typedef {import("coveo").SearchStatus} SearchStatus */
 /** @typedef {import("coveo").SearchEngine} SearchEngine */
+/**
```

Comment:
this typedef seems to be defined repeatedly. Could it be imported from a common place?

Reply:
The reason I didn't define this type in only one place and then import it where needed is because this type differs between the different facet types, it's not exactly the same..

Reply:
Ah my bad then I hadn't noticed the differences